runners/google-cloud-dataflow-java/arm/build.gradle

```diff
@@ -53,7 +53,7 @@ evaluationDependsOn(":sdks:java:container:java11")
 processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
     'dataflow.fnapi_environment_major_version' : '8',

```

Comment:
I think this should also be a reference. Not sure if it's used.

Reply:
good point, changed